packages/components/app/styles/mixins/_focus-ring.scss

```diff
@@ -8,7 +8,8 @@
 
   // default focus for browsers that still rely on ":focus"
   &:focus,
-  &.is-focus {
+  &.is-focus,
```

Comment:
once the changes done across all the pages, I'll remove the `is-[state]` selectors from here, for now we need both to avoid breaking the percy tests